contracts/ArchetypeGatekeep.sol

```diff
@@ -0,0 +1,653 @@
+// SPDX-License-Identifier: MIT
+// Archetype v0.6.0
```

Comment:
Can you add an identifier here, to distinguish it from vanilla Archetype. You can add your name too, no problem.

This is what we do for our 1155 contract, for example:
```
// SPDX-License-Identifier: MIT
// Archetype v0.6.0 - ERC1155
//
//        d8888                 888               888
//       d88888                 888               888
```